Source/WebCore/rendering/RenderBox.h

```diff
@@ -790,6 +804,9 @@ override;
     
     void applyTopLeftLocationOffsetWithFlipping(LayoutPoint&) const;
 
+    bool hasExplicitIntrinsicInnerWidth() const;
+    bool hasExplicitIntrinsicInnerHeight() const;
```

Comment:
It looks like these two methods are not needed, assuming they will only be used internally in RenderBox.cpp.

Reply:
Done, thanks!